### ISSUE TITLE
create export/import project feature 

### ISSUE THREAD

**Issue by patcmorneau** [CIDCO-dev/OpenSidescan#137]
export:
- create folder in path "opensidescan-executable-director/projectDir"
- copy "SSS files" in that directory
- file path are relative to "opensidescan-executable-director/projectDir"
- compress folder

import:
- the opposite of export

**Comment by patcmorneau** [comment]
poc : https://gitlab.com/mrpotatoqc9/compresseur

**Comment by Lecrapouille** [comment]
@patcmorneau ce serait bien d'avoir un README sur votre projet. J'ignore ce que vous voulez faire exactement mais tu peux utiliser le bon vieux .pak file :) https://quakewiki.org/wiki/.pak

**Comment by patcmorneau** [comment]
I don't about .pak but looks like the 56 maximum character for filename could create an issue for end user.
thanks for the share ;)
There is no official release yet.
The INSTALL.md file is well documented 
But for more information on The project you can have a look here : https://opensidescan.cidco.ca/

**Comment by Lecrapouille** [comment]
@patcmorneau yes was used by Quake 1 as zip-like storage at its time :) The site and work looks interesting. Could be nice working in Canada
